### PR TITLE
Add navigation and forms accessibility pages

### PR DESCRIPTION
**Pull request by guermanc**

### CODE REVIEW

File: apps/docs/src/common/pages/accessibility/forms.tsx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+<h3>Avoid using placeholders</h3>
+<p>Do not use placeholders that disappear when the field is being used, this makes it harder for people with memory issues and are not always read out by a screen reader. Using them instead of a label can make it hard for users to double check their answers. </p>
+
+<h3>Use hint text to help users</h3>
+<p>Hint text is additional information that helps users know what information to put in a field. Make sure it is short and to the point with examples of expected formats. Use <code>aria-describedby</code> to link the hint text to the field so that it is read out by screen readers. Don’t add links to hint text as screen readers won’t pick this up. It is also important to make sure that the hint text passes colour contrast.</p>
+<p>There is more advice about good hint text in the <A href="https://design-system.service.gov.uk/components/text-input/#hint-text">GOV.UK Design System</A></p>
+
+<div className="example" style={{marginBottom: "2em"}}>
+<div className="govuk-form-group">
+  <label className="govuk-label" for="national-insurance-number">
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/accessibility/forms.tsx:192]: `htmlFor=`

**Review by daniel-ac-martin — COMMENTED** [review]
LGTM.
@MarionDownie?

**Review by MarionDownie — COMMENTED** [review]
Forms page tweaks:

Separate into two sentences:
Forms are one of the main ways that we gather information from users. This can be anything from applying…

'This creates a better user experience' is jarring - better than what? I think it's better alone:
'This reduces errors that can cost money to address.'

I think we’ve had screenreader as one word elsewhere. Check all instances. 

Under 'Avoid using placeholders' - change from comma to dash
Do not use placeholders that disappear when the field is being used - this 

Under 'Use hint text to help users' 
Change Don’t to 'do not'

Full stop needed after 
GOV.UK Design System

**Comment by MarionDownie** [comment]
Comments for Navigation:

'some of the built-in screen reader functions to help users find their way around a page.' - need to replace to with that:
'some of the built-in screenreader functions that help users find their way around a page.'

Table
Motor impairment or impaired? Inconsistent across pages. Impaired probably fits better with partially sighted etc.

'Clear indicators of where they are in a journey and how many steps remain' - change to
'Clear indicators of where they are in a journey and how many steps are left'

**Comment by guermanc** [comment]
> I think we’ve had screenreader as one word elsewhere. Check all instances.

Correct terminology is screen reader and so I have changed this throughout the other pages.

**Pull request merged**